flake.nix

```diff
@@ -140,8 +136,18 @@
               installShellCompletion --bash "$(grep -oP 'RAGENIX_COMPLETIONS_BASH=\K.*' $buildOut)"
               installShellCompletion --zsh  "$(grep -oP 'RAGENIX_COMPLETIONS_ZSH=\K.*' $buildOut)"
               installShellCompletion --fish "$(grep -oP 'RAGENIX_COMPLETIONS_FISH=\K.*' $buildOut)"
+
+              # Symlink the plugins
+              for plugin in "${builtins.concatStringsSep " " plugins}"; do
+                ln -sf $plugin/bin/* $out/bin/
+              done
```

Comment:
Interesting approach! This wouldn't work, however, if the `ragenix` packages itself isn't in `PATH`. I'd suggest wrapping the resulting binary if any `plugins` are given. Please see the commit I added.